src/i18n/locales/da.json

```diff
@@ -5,6 +5,10 @@
     "female": "Kvinde",
     "male": "Mandlige",
     "total": "I alt",
+    "Welcome to The Home of Asylum Seekers": "Velkommen til Centralen for Asylansøgere",
```

Comment:
the keys for the json should be as short as possible. maybe rename this "title" since it's the title of the home page.

Reply:
From an organizational standpoint it's really helpful to nest all the text from one page into different JSON levels so that you can see all the text for one page clearly grouped together. this could look something like:
`{
    "homePage": {
        "title": "Welcome to The Home of Asylum Seekers",
        "aboutSection": {
            "title": "About Us"
            "text": "We could have something like this for the home page content with links and such. Just a bad looking idea, you are welcome to do something else :D"
        },
    },
    "nav": {
        "home": "Home",
        "summary": "Summary",
        "cases": "Cases",
    }
}`